### PR TITLE
Make Spark Thread pool configurable (#73)

### PR DESCRIPTION
**Pull request by finnWellers**

### CODE REVIEW

File: src/main/java/de/fearnixx/jeak/service/controller/controller/SparkAdapter.java [hunk 1]
```diff
@@ -19,16 +20,19 @@
 import java.util.List;
 import java.util.Map;
 
-import static spark.Spark.before;
-import static spark.Spark.halt;
+import static spark.Spark.*;
 
 public class SparkAdapter extends HttpServer {
     private static final Logger logger = LoggerFactory.getLogger(SparkAdapter.class);
+    public static final int MAX_THREADS = Main.getProperty("bot.sparkadapter.maxpoolsize", 8);
```
Comment (MarkTwoFive) [inline src/main/java/de/fearnixx/jeak/service/controller/controller/SparkAdapter.java]: I would prefer having something a little more generic. Like ``jeak.http.poolsize``.  
Having min and max possible as separate values is nice to have but we definitively need an option to set both at the same time for ordinary users.  
If you want to, you could still offer the Spark-specific property by reading both a generic and a specific option but definitely use the ?``jeak.``-prefix then.

Comment (finnWellers) [reply src/main/java/de/fearnixx/jeak/service/controller/controller/SparkAdapter.java]: With both you mean configure a fixed value? Spark only offers a max-threads and a max and min threads constructor. Of course I could pass one value as value for both.

Comment (MarkTwoFive) [reply src/main/java/de/fearnixx/jeak/service/controller/controller/SparkAdapter.java]: The updated implementation looks good to me as it is.

**Comment by MarkTwoFive** [comment]
Please merge the current bleeding branch or rebase the PR for the Snyk check to pass.  
I believe they're detecting the already updated dependencies as vulnerabilities even though they would not be changed back.

**Review by MarkTwoFive — APPROVED** [review]
Thank you for the contribution :)

**Pull request merged**